sources/api/apiserver/src/server/ephemeral_storage.rs

```diff
@@ -298,6 +338,29 @@ pub fn ephemeral_devices() -> Result<Vec<String>> {
     Ok(filenames)
 }
 
+/// ephemeral_ebs_volumes returns the full path name to the ebs volumes in /dev/disk/ephemeral-ebs
+pub fn ephemeral_ebs_volumes() -> Result<Vec<String>> {
+    const EPHEMERAL_EBS_PATH: &str = "/dev/disk/ephemeral-ebs";
```

Comment:
nit: I'd prefer this defined at the top of this file as `static` below `EPHEMERAL_STORAGE_LINK`